cmd/syncpending.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"github.com/juju/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/bitnami-labs/charts-syncer/api"
+	"github.com/bitnami-labs/charts-syncer/pkg/config"
+	"github.com/bitnami-labs/charts-syncer/pkg/syncer"
+)
+
+var (
+	syncPendingFromDate string
+)
+
+var (
+	syncPendingExample = `
+  # Synchronizes all charts defined in the configuration file
+  charts-syncer sync
```

Comment:
`sync` -> `sync-pending`

Reply:
> What's the difference with the current `sync` command?
> 
> The `sync` one iterates over dependencies as they are found and this one uses the topological sort first?

I am just leaving `sync` intact while I work on the new sync under `sync-pending`. Once all of this is done, we will only have `sync`.

In any case, yes, this new method takes care of getting a list of charts topologically sorted and sync then without worrying about dependencies anymore.